components/director/internal/domain/labeldef/resolver.go

```diff
@@ -124,3 +126,38 @@ func (r *Resolver) LabelDefinition(ctx context.Context, key string) (*graphql.La
 	c := r.conv.ToGraphQL(*def)
 	return &c, nil
 }
+
+func (r *Resolver) UpdateLabelDefinition(ctx context.Context, in graphql.LabelDefinitionInput) (*graphql.LabelDefinition, error) {
+	tnt, err := tenant.LoadFromContext(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	tx, err := r.transactioner.Begin()
+	if err != nil {
+		return nil, errors.Wrap(err, "while starting transaction")
+	}
+	defer r.transactioner.RollbackUnlessCommited(tx)
+
+	ctx = persistence.SaveToContext(ctx, tx)
+
+	// TODO: Use LabelDefinitionInput
+	ld := r.conv.FromGraphQL(in, tnt)
+
+	err = r.srv.Update(ctx, ld)
+	if err != nil {
+		return nil, errors.Wrap(err, "while updating label definition")
+	}
+	if err := tx.Commit(); err != nil {
+		return nil, errors.Wrap(err, "while committing transaction")
+	}
+
+	updatedLd, err := r.srv.Get(ctx, tnt, in.Key)
```

Comment:
This Get will always return error because the transaction is already committed at line 151.

Reply:
done